layouts/_default/baseof.html

```diff
@@ -88,7 +88,7 @@
         <link rel="alternate" type="application/rss+xml" title="Istio Blog and News" href="/feed.xml">
 
         <!-- Favicons: generated from img/istio-whitelogo-bluebackground-framed.svg by http://cthedot.de/icongen -->
-        <link rel="shortcut icon" href="/favicons/favicon.ico" >
+        <link rel="shortcut icon" href="/favicon.ico" >
```

Comment:
Both seem to be the same file?

Reply:
I deleted the extra file, there is the same
https://github.com/istio/istio.io/pull/10187/files#diff-662d8b2c61bb9159272d9b8f1864ebaf4bbb5cb83a30cde1c7afbba602d48db7
I updated it

Reply:
Would it not be best to keep favicon.ico in /favicons ?  We can map a redirect for /favicon.ico if we need to.

Reply:
Ok

Reply:
try adding

```/favicon.ico /latest/favicons/favicon.ico 200```

to `layouts/index.redir` #46